src/uwtools/drivers/fv3.py

```diff
@@ -217,33 +217,10 @@ def runscript(self):
 
     # Private workflow tasks
 
-    @external
-    def _file(self, path: Path):
-        """
-        An existing file.
-
-        :param path: Path to the file.
-        """
-        yield "File %s" % path
-        yield asset(path, path.is_file)
-
-    @task
-    def _filecopy(self, src: Path, dst: Path):
-        """
-        A copy of an existing file.
-
-        :param src: Path to the source file.
-        :param dst: Path to the destination file to create.
-        """
-        yield "Copy %s -> %s" % (src, dst)
-        yield asset(dst, dst.is_file)
-        yield self._file(src)
-        copy(src, dst)
-
     @task
     def _run_via_batch_submission(self):
         """
-        FV3 run Execution via the batch system.
+        Run execution via the batch system.
```

Comment:
It should be a noun, not a verb: Tasks are stand-ins for what they create (asset(s)), and this task creates "a run". Would
> A run executed via the batch system.

be better? I think it's important to avoid thinking of tasks as actions, or using language that obscures their asset-orientation. Focusing on nouns, the question "What exactly IS a 'run'?" comes to mind, and it's a crucial question to have a good answer to. Some of the outputs/assets wrapped by tasks will necessarily be somewhat conceptual, but there will have to be some defined, observable state -- maybe an enumerated set of expected output files, or even just a sentinel file written at the end of the run.

Reply:
I went ahead and updated these task descriptions. We can certainly revisit if there are better ideas later.